package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mozilla-frontend-infra/perf-goggles",
-  "version": "2.0.1",
+  "version": "3.0.0",
```

Comment:
Don't change the version yet. There will be more changes in this release.
Is this a breaking change?